Rakefile

```diff
@@ -5,29 +5,24 @@ require 'mechanize'
 require 'fileutils'
 require 'yaml'
 require 'launchy'
-
-CDK_DOWNLOAD_URL='https://access.redhat.com/downloads/content/293/ver=2/rhel---7/2.0.0/x86_64/product-software'
-CDK_BOX_BASE_NAME='rhel-cdk-kubernetes-7.2-23.x86_64.vagrant'
-
-CDK_DOWNLOAD_URL_NIGHTLY='http://cdk-builds.usersys.redhat.com/builds/nightly/latest-build'
-
-ADB_DOWNLOAD_URL='http://cloud.centos.org/centos/7/atomic/images'
-ADB_BOX_BASE_NAME='AtomicDeveloperBundle-2.1.0-CentOS7'
+require_relative 'download_task'
```

Comment:
I thought you would like that :-)
The Rakefile is then focused on the actual build tasks.